starfish/test/experiment/test_experiment.py

```diff
@@ -1,17 +1,90 @@
+from typing import Tuple
+
+import numpy as np
+from slicedimage import Tile, TileSet
+
 import starfish.data
 from starfish.experiment.experiment import Experiment, FieldOfView
-from starfish.imagestack.imagestack import ImageStack
-from starfish.imagestack.parser.crop import CropParameters
+from starfish.types import Axes, Coordinates
 from starfish.util.synthesize import SyntheticData
 
 
+def round_to_x(r: int) -> Tuple[float, float]:
+    return (r + 1) * 1000, (r + 1) * 100
+
+
+def round_to_y(r: int) -> Tuple[float, float]:
+    return (r + 1) * 10, (r + 1) * 0.1
+
+
+def round_to_z(r: int) -> Tuple[float, float]:
+    return (r + 1) * 0.01, (r + 1) * 0.001
+
+
+NUM_ROUND = 5
+NUM_CH = 2
+NUM_Z = 1
+HEIGHT = 10
+WIDTH = 10
+
+
+def get_aligned_tilset():
+    alignedTileset = TileSet(
+        [Axes.X, Axes.Y, Axes.CH, Axes.ZPLANE, Axes.ROUND],
+        {Axes.CH: NUM_CH, Axes.ROUND: NUM_ROUND, Axes.ZPLANE: NUM_Z},
+        (HEIGHT, WIDTH))
+
+    for r in range(NUM_ROUND):
```

Comment:
😆 It would take me 5 minutes to figure out the way that this isn't aligned by following what `round_to_z` is doing. Could you add a comment? 

Also I thought, based on above, that we're only testing alignment on (x, y) -- but here it looks like `(x, y)` are all the same and we're testing alignment over z? I'm very confused!

Reply:
we're testing that z is set to the midpoint of each zrange, like discussed

Reply:
They should all vary because they all get multiplied based on the round value. But I added a comment!